Pipfile.lock

```diff
@@ -74,6 +74,13 @@
                 "sha256:d5f05e487007e29e03409f9398d074e158d920d36eb82eaf66fb1136b0c5374c"
             ],
             "version": "==2018.9"
+        },
+        "pyyaml": {
```

Comment:
It's annoying that we have to update one of our dependency's dependencies, but probably the right call. That said, since before `pyyaml` only shows up in the `dev` part of the lock file, I think you can just `pipenv install --dev pyyaml`, so it's only listed as a dev dependency and won't be built on our production server at all.

Reply:
Oi, this is more challenging to fix than I thought. The issue with doing a --dev install is that awsebcli needs 3.10-3.13 to work, so installing 4.* pyyaml conflicts with that.

Reply:
Well that's annoying. I might just ignore it then, and wait for aws to update its dependency requirement.